### PR TITLE
Added score logging for Atari

### PR DESCRIPTION
**Pull request by KhrulkovV**
Added score tracking (total unclipped reward) for the Atari envs.

### CODE REVIEW

File: examples/atari/src/atari_wrappers.py [hunk 1]
```diff
@@ -304,6 +305,85 @@ def __getitem__(self, i):
...
+        self.total_steps = 0
+        self.current_reset_info = {}
+
+    def reset(self, **kwargs):
+        self.reset_state()
+        for k in self.reset_keywords:
+            v = kwargs.get(k)
+            if v is None:
+                raise ValueError(
+                    'Expected you to pass kwarg %s into reset' % k
```
Comment (Scitator) [inline examples/atari/src/atari_wrappers.py:335]: please use " to follow lib standards

File: examples/atari/src/atari_wrappers.py [hunk 2]
```diff
@@ -304,6 +305,85 @@ def __getitem__(self, i):
...
+            eplen = len(self.rewards)
+            epinfo = {"r": round(eprew, 6), "l": eplen}
+            for k in self.info_keywords:
+                epinfo[k] = info[k]
+            self.episode_rewards.append(eprew)
+            self.episode_lengths.append(eplen)
+            epinfo.update(self.current_reset_info)
+            assert isinstance(info, dict)
+            if isinstance(info, dict):
+                info['episode'] = epinfo
```
Comment (Scitator) [inline examples/atari/src/atari_wrappers.py:370]: "

**Comment by Scitator** [comment]
replaced by https://github.com/catalyst-team/catalyst/pull/247?

**Comment by KhrulkovV** [comment]
Indeed, replaced by #247.

**Pull request closed without merging**